tests/unit/test.program.js

```diff
@@ -32,7 +36,9 @@ describe('program.Program', () => {
     let thing = spy(() => Promise.resolve());
     let program = new Program(['thing'])
       .command('thing', 'does a thing', thing);
-    return execProgram(program)
+    return execProgram(program, {
+      checkForUpdates: spy(),
```

Comment:
Maybe it wasn't defined in the right part of `execProgram()`? If it's not working, post a broken patch and I can take a look.